docs/changelog/1274.md

```diff
@@ -1 +1 @@
-- Renamed class 'MasterSlave' to 'IntraComm' and changed all corresponding names. Changed the XML tag 'master' to 'intra-comm' and changed the XML attribute 'on-slaves' to 'on-secondary-rank'.
+- Renamed class 'MasterSlave' to 'IntraComm' and changed all corresponding names. Changed the XML tag 'master' to 'intra-comm' and changed the XML attributes 'on-slaves' and 'on-master' to 'on-secondary-rank' and 'on-primary-rank' respectively. The tag 'master' and attributes 'on-slaves' and 'on-master' are deprecated.
```

Comment:
`on-secondary-ranks`, right? not `on-secondary-rank`?

Reply:
Fixed in https://github.com/precice/precice/pull/1283